src/transformers/models/jukebox/configuration_jukebox.py

```diff
@@ -539,8 +539,6 @@ class JukeboxConfig(PretrainedConfig):
         metadata_conditioning (`bool`, *optional*, defaults to `True`):
             Whether or not to use metadata conditioning, corresponding to the artist, the genre and the min/maximum
             duration.
-        init_std (`float`, *optional*, defaults to 0.2):
-            Standard deviation used to initial the model.
```

Comment:
@ArthurZucker Could you check this unused attribute?

Reply:
Jukebox is unused 😅 But yes this argument is not used